lib/controller/controller.go

```diff
@@ -332,44 +329,6 @@ func RetryPeriod(retryPeriod time.Duration) func(*ProvisionController) error {
 	}
 }
 
-// ClaimsInformer sets the informer to use for accessing PersistentVolumeClaims.
```

Comment:
we shouldn't remove these. someone out there is using them and anyway it's in the spirit of sharedinformers to allow users to pass them in: theoretically they would pass sharedinformers into this controller + whatever other controllers are also using the sharedinformer and then start the informers whenever they want.
hope that makes sense.
Don't get me wrong, I would like to clean up the code as much as possible but we have to maintain support for the two cases of: 1) user provides their own informers, 2) we create and run them ourselves

Reply:
> whatever other controllers are also using the sharedinformer and then start the informers whenever they want.

I think this is the main reason. Since we all using `SharedInformers`, no matter where we create the informer, we are using the same instance of the cache as everyone else.